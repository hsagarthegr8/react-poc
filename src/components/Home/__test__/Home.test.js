import Home from '../Home'

describe("Home Component", () => {
    test("it should match Snapshot", () => {
        const wrapper = mount(<Home />)
        expect(wrapper).toMatchSnapshot()
    })
})