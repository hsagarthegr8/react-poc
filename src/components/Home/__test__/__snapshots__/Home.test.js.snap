// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component it should match Snapshot 1`] = `
<Home>
  <div>
    Home
  </div>
</Home>
`;
